acceptance_tests/features/inbox_internal.feature

```diff
@@ -36,3 +36,21 @@ Feature: Internal inbox
     Given the user has access to secure messaging
     When they navigate to the select survey page
     Then they are able to view a list of filter options
+
+  @sm114_s01
+  Scenario: User is able to distinguish unread messages in their inbox
+    Given the user has no messages in their inbox
+    And the user has an unread message in their inbox
```

Comment:
It'd be impossible for the user to have no messages AND an unread message.  Might be better to switch the logic around a bit to something along the lines of:

Given the user is signed in
And have access to secure messaging
And the user has no messages in their inbox
When the user receives a message from an external user
And the user navigates to their inbox
Then the user is able to see 1 unread message in their inbox

Similar for scenario @sm114_s02